src/syslog_mcp/services/elasticsearch_client.py

```
@@ -97,6 +97,9 @@ class ElasticsearchConfig(BaseSettings):
     connection_max_age: float = Field(default=300.0, alias="ELASTICSEARCH_CONNECTION_MAX_AGE") # 5 minutes
     enable_connection_pooling: bool = Field(default=True, alias="ELASTICSEARCH_ENABLE_POOLING")
     
+    # Index configuration  
```

Ruff (0.12.2)

Trailing whitespace

Remove trailing whitespace

(W291)